source/core/parameters_lagrangian.cc

```diff
@@ -320,6 +320,31 @@ namespace Parameters
                           "0",
                           Patterns::List(Patterns::Double()),
                           "List of particles z positions");
+
+        prm.declare_entry("velocity x",
+                          "0.0",
+                          Patterns::Double(),
+                          "Initial velocity x");
+        prm.declare_entry("velocity y",
+                          "0.0",
+                          Patterns::Double(),
+                          "Initial velocity y");
+        prm.declare_entry("velocity z",
+                          "0.0",
+                          Patterns::Double(),
+                          "Initial velocity z");
+        prm.declare_entry("omega x",
+                          "0.0",
+                          Patterns::Double(),
+                          "Initial omega x");
+        prm.declare_entry("omega y",
+                          "0.0",
+                          Patterns::Double(),
+                          "Initial omega y");
+        prm.declare_entry("omega z",
+                          "0.0",
+                          Patterns::Double(),
```

Comment:
This means that all particles are inserted with the same values of the velocity and omega right?

Reply:
They are. I don't know how I could separate them

Reply:
``Insertion info`` could be divided according to ``particle type``, but I can't think of a situation in which this would be important

Reply:
For the time being this is ok as a mechanism I think. The current input format make it difficult to have very flexible insertion mechanism.